soc/arm/gigadevice/Kconfig

```diff
@@ -20,6 +20,11 @@ config GD32_HAS_AF_PINMUX
 	help
 	  This option should be selected if the series use an AF pinmux model.
 
+config GD32_HAS_AFIO_PINMUX
+	bool
+	help
+	  This option should be selected if the series use an AFIO pinmux model.
+
```

Comment:
Same as `GD32_HAS_AF_PINMUX`, better move to modules/haL_gigadevice

Reply:
done, makes sense to cover RISC-V